frontend/src/app/pages/exercises/exercise/shared/simulation/signaller-modal/signaller-modal-region-information/signaller-modal-region-information.component.ts

```diff
@@ -0,0 +1,363 @@
+import type { OnChanges, OnInit } from '@angular/core';
+import { Component, Input, TemplateRef, ViewChild } from '@angular/core';
+import { Store } from '@ngrx/store';
+import type { ReportableInformation } from 'digital-fuesim-manv-shared';
+import type { UUID } from 'digital-fuesim-manv-shared';
+import { makeInterfaceSignallerKey } from 'digital-fuesim-manv-shared';
+import { type Observable, BehaviorSubject, map } from 'rxjs';
+import { ExerciseService } from 'src/app/core/exercise.service';
+import { Hotkey } from 'src/app/shared/services/hotkeys.service';
+import type { AppState } from 'src/app/state/app.state';
+import { selectOwnClientId } from 'src/app/state/application/selectors/application.selectors';
+import { selectStateSnapshot } from 'src/app/state/get-state-snapshot';
+import { createSelectBehaviorStates } from 'src/app/state/application/selectors/exercise.selectors';
+import {
+    setLoadingState,
+    type InterfaceSignallerInteraction,
+} from '../signaller-modal-interactions/signaller-modal-interactions.component';
+import { SignallerModalDetailsService } from '../details-modal/signaller-modal-details.service';
+
+@Component({
+    selector: 'app-signaller-modal-region-information',
+    templateUrl: './signaller-modal-region-information.component.html',
+    styleUrls: ['./signaller-modal-region-information.component.scss'],
+    standalone: false,
+})
+export class SignallerModalRegionInformationComponent
+    implements OnInit, OnChanges
+{
+    @Input()
+    simulatedRegionId!: UUID;
+
+    @ViewChild('recurringReportEditor')
+    recurringReportEditor!: TemplateRef<any>;
+
+    informationTypeToEdit: ReportableInformation | null = null;
+
+    private clientId!: UUID;
+
+    informationInteractions: InterfaceSignallerInteraction[] = [
+        {
+            key: 'patientCount',
+            name: 'Anzahl Patienten',
+            details: 'nach Sichtungskategorie',
+            keywords: [
+                'patient',
+                'patienten',
+                'zahl',
+                'anzahl',
+                'zahlen',
+                'zählen',
+                'menge',
+            ],
+            hotkey: new Hotkey('A', false, () => this.requestPatientCount()),
+            secondaryHotkey: new Hotkey('⇧ + A', true, () =>
+                this.openRecurringReportModal('patientCount')
+            ),
```

Comment:
I feel like some kind of unification would be good here. Currently, the modal type is declared twice, once in the secondary hotkey and once in the function called by the hotkey. Also, these calls are more or less identical between the different interactions. However, I'm also not sure how this could be unified in a reasonable way.

Reply:
Like with the other comment, I agree that I don't see a nice way to simplify this, so keep it like it is.